src/Orleans/Core/InterceptedMethodInvoker.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using Orleans.CodeGeneration;
+using Orleans.Runtime;
+
+namespace Orleans
+{
+    /// <summary>
+    /// An <see cref="IGrainMethodInvoker"/> which redirects execution for grains which implement
+    /// <see cref="IGrainInvokeInterceptor"/>.
+    /// </summary>
+    internal class InterceptedMethodInvoker : IGrainMethodInvoker
```

Comment:
This class was shifted from another file & should not be functionally different to how it was with #965